tests/conftest.py

```diff
@@ -185,7 +185,7 @@ def pv_yields_and_systems(db_session):
             ).to_orm()
             if datetime_utc.hour in [22, 23, 0, 1, 2]:
                 pv_yield_1.solar_generation_kw = 0
-            pv_yield_1.pv_system = pv_system_sql_1
+            pv_yield_1.pv_system = pv_system_sql_1s
```

Comment:
change this back?

Reply:
Ah okay, yeah, I was wondering why tests were failing on the PV systems, thanks!